### ISSUE TITLE
Bug after recent Tensorflow update

### ISSUE THREAD

**Issue by adai183** [udacity/CarND-LeNet-Lab#16]
`cross_entropy = tf.nn.softmax_cross_entropy_with_logits(logits, one_hot_y)`
need to be:
`cross_entropy = tf.nn.softmax_cross_entropy_with_logits(logits=logits, labels=one_hot_y)`

If not we get:

`Raise en error if labels do not sum to 1` exception

This came up in the discussion forum.

**Comment by mwhayford** [comment]
Thanks for this comment! I had this same issue to when loading for the first time.

**Comment by nandanlahurikar** [comment]
cross_entropy = tf.nn.softmax_cross_entropy_with_logits(logits=LeNet(x), labels=one_hot_y)

**Comment by zippau** [comment]
agree... cross_entropy = tf.nn.softmax_cross_entropy_with_logits(logits=logits, labels=one_hot_y)

**Comment by CreatCodeBuild** [comment]
I am a classroom mentor. New students start to get this bug because they use 1.0.

I think Udacity code should use 1.0 standard as well.

**Comment by shaox058** [comment]
Hi @CreatCodeBuild @adai183 @mwhayford 

I have the same issue here. Should I downgrade my tensorflow? If yes, which version is the best fit.

Thanks,
Harry

**Comment by CreatCodeBuild** [comment]
@shaox058, the best way it to modify your code. It's trivial to do so. You
just need keyword arguments.

**Comment by mvirgo** [comment]
Since we are still using v0.12.1, closing this - will be updated when we move to the newer versions.
